spec/features/create_photos_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature 'Photo' do
+  it 'can create a photo' do
+    visit 'photos/new'
+    attach_file('photo_image', Rails.root + 'spec/files/test.png', visible: false)
+    click_button 'Upload'
+    expect(Photo.count).to eq 1
```

Comment:
small code smell here: checking the exact count is brittle. You can get around it by counting the current photos at the beginning of the test and then expecting that the count increases.